hw-model/src/model_fpga_subsystem.rs

```diff
@@ -526,6 +644,18 @@ impl ModelFpgaSubsystem {
             }
         }
 
+        let status = self
+            .i3c_core()
+            .sec_fw_recovery_if()
+            .recovery_status()
+            .read()
+            .dev_rec_status();
+        if status == 3 {
```

Comment:
nit: Similar enum comment here, not sure what `3` means at a glance, but can be refactored later